README.md

```diff
@@ -33,32 +37,31 @@ resources:
     params:
       driver: rabbitmq
       datasource: amqp://guest:guest@localhost:5672
-      
 ```
 
-[List of available resources](http://alileza.github.io/tomato/resources)
-
-Once you're ready with all your resources you needed, you're good to start writing your features. You can find some of examples [here](https://github.com/alileza/tomato/tree/0.1.0/examples/features)
+Once your tomato.yml is set up with all the resources your application requires, you can begin writing features. Features are pre-defined behavior tests that are run against your resources to determine if the application performs as expected. You can find some examples [here](https://github.com/alileza/tomato/tree/0.1.0/examples/features)
 
 `example.feature`
 ```gherkin
-Feature: behavior test example
-
-  Scenario: Delete customer on DELETE http request
-    Given listen message from "my-awesome-queue" target "customers:deleted"
-    Given set "my-awesome-postgres-db" table "customers" list of content
-        | name    | country |
-        | cembri  | id      |
-    Given "httpcli" send request to "DELETE /api/v1/customers" with body
+Feature: Customer removal via an HTTP API request
+
+  # Set the stage for the test
+  Background:
+    Given the table "customers" in "my-awesome-postgres-db" with content
+      | name    | country |
+      | cembri  | id      |
+
+  Scenario: Remove cembri when a rabbitmq delete message is received
+    Given a message received from "my-awesome-queue" target "customers:deleted"
+    When the "httpcli" sends a DELETE request to "/api/v1/customers" with body
```

Comment:
I think naming `httpcli` can be a bit misleading, I'm thinking to use service name instead. Since most of the time we specify base url, so http client would always being used for the same service. So, `my-service-client`, what do you think?

Also having example of resource file right above it might be helpful to give a view that "httpcli" is referring to resource file

Reply:
Agreed, it makes it more obvious what the resource is